### PR TITLE
fix: disable QR camera

### PR DESCRIPTION
**Pull request by sebastianquek**
The QR camera currently keeps scanning even though the user has navigated to the scanned document screen.

Also added `ratio` prop for the camera as the preview was getting stretched on Android.

### CODE REVIEW

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=34

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-34

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=34

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-34

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=34

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-34

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=34

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-34

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=34

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-34

File: src/components/QrScanner/QrScannerScreenContainer.tsx [hunk 1]
```diff
@@ -25,15 +25,31 @@ export const QrScannerScreenContainer: FunctionComponent<QrScannerScreenContaine
...
   };
   const onQrData = async (data: string): Promise<void> => {
+    if (scanningDisabled) {
+      return;
+    }
     setScanningDisabled(true);
     try {
       await processQr(data, { onDocumentStore, onDocumentView });
     } catch (e) {
+      setScanningDisabled(false);
```
Comment (yehjxraymond) [inline src/components/QrScanner/QrScannerScreenContainer.tsx]: Think this needs to be outside of the try catch, if not after if there is a case where the processQr does not lead to this screen being unmounted, the `scanningDisabled` will still be true

Comment (sebastianquek) [reply src/components/QrScanner/QrScannerScreenContainer.tsx]: Hmm, so you're saying that `processQr` succeeds but doesn't cause a navigation?

Comment (yehjxraymond) [reply src/components/QrScanner/QrScannerScreenContainer.tsx]: Yep!

Comment (sebastianquek) [reply src/components/QrScanner/QrScannerScreenContainer.tsx]: As spoken, given that `processQr` will either cause a navigation or throw an Error, we'll leave this in. The other issue with moving it out was that it gets called after the screen blurs, which makes the qr scanning work when we don't want it to.

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=34

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-34

**Pull request merged**